spec/controllers/api/v1/tasks_controller_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::TasksController, type: :controller do
+  let(:tasks) { create_list(:task, 10) }
+
+  # index action
+  describe 'GET index' do
+    subject { get :index, format: :json }
+
+    it 'status should be' do
+      expect(subject.status).to eq(200)
+    end
+
+    it 'when there are not tasks' do
```

Comment:
@JoseCastroA I think this example should be in a different context, something like the following
```ruby
describe 'GET index' do
  context 'when there are not tasks' do
    # this example.
  end

  context 'when there are tasks' do
    let!(:tasks) { .... }

    # Examples for success case.
  end
end
```